src/util/CommandLine.ts

```diff
@@ -24,4 +24,14 @@ export function commandExecutor(scriptParamName: string) {
     const script = getParameter(scriptParamName);
     if (script === undefined) return undefined;
     return () => console.log(`${scriptParamName}: ${execSync(script).toString().slice(0, -1)}`);
-}
\ No newline at end of file
+}
+
+export function getFloatCommandResult(script: string) {
+    return (parseFloat(`${execSync(script).toString().slice(0, -1)}`) || 0);
```

Comment:
I don't think we want to return 0 if we cannot parse the float since this will swallow issues, making them hard to debug.
Instead, throw an Error:
```
const commandResult = executeCommand(script);
const parsedResult = parseFloat(commandResult);
if (parsed === NaN) throw new Error(`Cannot parse a float from: ${commandResult}`);
```

Same below

Reply:
Agree. During debug was just concerned with getting any value, but not appropriate for production. Will change.